### ISSUE TITLE
Refactor: SidebarLeftFooter component to having props for maintainability and readability

### ISSUE THREAD

**Issue by luuu-xu** [activist-org/activist#374]
### Terms

- [X] I have searched [open and closed feature requests](https://github.com/activist-org/activist/issues?q=is%3Aissue+label%3Afeature)
- [X] I agree to follow activist's [Code of Conduct](https://github.com/activist-org/activist/blob/main/.github/CODE_OF_CONDUCT.md)

### Description

For now, our `SidebarLeftFooter` component's structure is complex and redundant, preferably it should be refactored into a similar structure as `SidebarLeftIndex` component, with:

1. Reduce redundant repetitions of smaller components.
2. Include in-component props data for maintainability like this:
```
const organizationButtons: SidebarLeftSelectorType[] = [
  {
    label: "components.sidebar-left-selector.label.about",
    routeURL: "/organizations/" + id + "/about",
    iconURL: "bi:card-text",
    selected: useRoute().path.split("/").pop() === "about" ? true : false,
    active: true,
  },
  {
    label: "components.sidebar-left-selector.label.events",
    routeURL: "/organizations/" + id + "/events",
    iconURL: "bi:calendar-check",
    selected: useRoute().path.split("/").pop() === "events" ? true : false,
    active: true,
  },
```
3. Change how i18n `$t()` function is used in the component, it should follow `SidebarLeftIndex` where we are passing in i18n key (like `:label="components.sidebar-left-selector.label.about"` into `SidebarLeftSelector` rather than the current way `:label=$t('components.sidebar-left-selector.label.new-event')"`.

### Contribution

I will be working on this issue.

**Comment by andrewtavis** [comment]
Thanks for the issue, @luuu-xu! Looking forward to the improvements :)

**Comment by andrewtavis** [comment]
0c7b7fd does the minor fixes that I mentioned in the PR, @luuu-xu. The legal icon was fixed, and I also formatted a separate component to the one in this issue that was causing the formatting check to fail. Thanks for the work here, and sorry that I jumped into the i18n work for point 3 above! I got into a bit of a groove when I was working on it and the full context of this issue slipped my mind.
